hirte.spec.in

```diff
@@ -145,6 +155,13 @@ This package contains the service controller command line tool.
 %{_bindir}/hirtectl
 %{_mandir}/man1/hirtectl.*
 
+%files -n python3-%{name}
+%license LICENSE
+%{python3_sitelib}/%{name}/__init__.py
+%{python3_sitelib}/%{name}/__pycache__/__init__.*.pyc
+%{python3_sitelib}/%{name}/config.py
+%{python3_sitelib}/%{name}/__pycache__/config.*.pyc
```

Comment:
Based on 
https://docs.fedoraproject.org/en-US/packaging-guidelines/Python/#_dist_info_metadata
the meta data file is required.

Reply:
just include the entire directory `__pycache__`